Atomic/install.py

```diff
@@ -49,17 +49,12 @@ def cli(subparser):
         help=_("preview the command that %s would execute") % sys.argv[0])
     installp.add_argument("image", help=_("container image"))
     if OSTREE_PRESENT:
-        bwrap_oci_available = util.bwrap_oci_available()
-        runc_available = util.runc_available()
-        if bwrap_oci_available or runc_available:
-            system_xor_user = installp.add_mutually_exclusive_group()
-            if bwrap_oci_available:
-                system_xor_user.add_argument("--user", dest="user", action="store_true", default=False,
-                                             help=_("Flag to specify if user is non-root privileged."))
-            if runc_available:
-                system_xor_user.add_argument("--system", dest="system",
-                                             action='store_true', default=False,
-                                             help=_('install a system container'))
+        system_xor_user = installp.add_mutually_exclusive_group()
+        system_xor_user.add_argument("--user", dest="user", action="store_true", default=False,
+                                     help=_("Flag to specify if user is non-root privileged."))
```

Comment:
Shouldn't this be soemthing like

Install the image as a user image?  Or a non-root image?